server/core/src/main/scala/sttp/tapir/server/metrics/Metric.scala

```diff
@@ -64,3 +64,22 @@ object MetricLabels {
     )
   )
 }
+
+object OTELMetricLabels {
+
+  /** Labels request by path and http.request.method, response by http.response.status_code */
+  lazy val Default: MetricLabels = MetricLabels(
+    forRequest = List(
+      "http.request.method" -> { case (_, req) => req.method.method },
```

Comment:
do any of the optional attributes make sense to include?
also, can you add a comment pointing to the otel docs where the attributes are specified?